src/app/shared/services/records/review-note.service.ts

```diff
@@ -24,11 +24,13 @@ export class ReviewNoteService extends GenericListService {
     return this.list<PaginatedResponse<ReviewNote>>('review-notes', req);
   }
 
-  createReviewNote(dres: Record, notecontent: string): Observable<ReviewNote> {
-    const data = new FormData();
-    data.set('content', notecontent);
-    data.set('record', dres.id.toString());
-    data.set('user', dres.user.toString());
+  createReviewNote(dres: Record, notecontent: string, userID: number, username: string): Observable<ReviewNote> {
```

Comment:
What does `dres` mean? The backend API for creating `ReviewNote` should only accept `record` and `content` as parameters, others should be deleted.